src/main/java/cceclipseplugin/core/PluginManager.java

```diff
@@ -511,7 +515,8 @@ private void initPropertyListeners() {
 							autoSubscribeForSession = false;
 							removeAllSubscribedPrefs(false);
 						}
-						System.out.println("Auto-subscribe for session set to " + autoSubscribeForSession);
+						Logger.getInstance().log(IStatus.WARNING,
```

Comment:
Same with this class. Pull out the result of `Logger.getInstance()` into a field?